### ISSUE TITLE
[vscode] [slack] Invalid token

### ISSUE THREAD

**Issue by daveek** [karigari/vscode-chat#245]
This doesn't allow me to connect Slack

- Extension Version: 0.7.3
- OS Version: Windows_NT x64 6.1.7601
- VSCode version: 1.28.1

**Comment by arjun27** [comment]
thanks for reporting this @daveek. Did you try the "Sign in with Slack" command? It will configure the slack token automatically, and is more reliable. Let me know if there are any issues with doing that

**Comment by daveek** [comment]
Exactly, I followed the instructions in order to get the connection using the token generated... but It says: "It was unable to validate the Token".

**Comment by arjun27** [comment]
does the token start with `xoxp-...`? Unfortunately the token input does not sanitize the input for whitespaces or quotation marks right now

**Comment by daveek** [comment]
![reported-issue](https://user-images.githubusercontent.com/4696050/47073787-5650d500-d1f9-11e8-8523-3b4775389097.png)

**Comment by daveek** [comment]
Yes, the token start with `xoxp-4`

**Comment by arjun27** [comment]
Hmm, I'm checking if this will fail for a valid token. Can you try the "Sign in with Slack" command instead?

**Comment by arjun27** [comment]
Hey @daveek - any luck with this? We have compiled a list of possible solutions to slack auth here: https://github.com/karigari/vscode-chat/blob/master/docs/SLACK.md

Would any of these work for you?
